### PR TITLE
[RW-7505][risk=no] Separate validate command for adding prep surveys to cdr build

### PR DESCRIPTION
**Pull request by dolbeew**

### CODE REVIEW

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -847,6 +847,44 @@ def make_cb_criteria(cmd_name, *args)
...
+    "--data-browser [data-browser]",
+    ->(opts, v) { opts.data_browser = v},
+    "Is this run for data browser. Optional - Default is false"
+  )
+  op.add_validator ->(opts) { raise ArgumentError unless opts.bq_project and opts.bq_dataset and opts.cdr_version }
+  op.parse.validate
+
+  common = Common.new
+  Dir.chdir('db-cdr') do
+    common.run_inline %W{./generate-cdr/validate-prerequisites-exist.sh #{op.opts.bq_project} #{op.opts.bq_dataset} #{op.opts.cdr_version} #{op.opts.data_browser}}
```
Comment (freemabd) [inline api/libproject/devstart.rb:878]:
```suggestion
    common.run_inline %W{./generate-cdr/validate-prerequisites-exist.sh #{op.opts.bq_project} #{op.opts.bq_dataset} #{op.opts.data_browser}}
```

**Review by freemabd — COMMENTED** [review]
Few comments

File: api/db-cdr/generate-cdr/validate-prerequisites-exist.sh [hunk 2]
```diff
@@ -8,6 +8,7 @@ export BQ_DATASET=$2        # CDR dataset
 export CDR_VERSION=$3       # CDR version
```
[the remark this answers is not in the document]
Comment (dolbeew) [reply api/db-cdr/generate-cdr/validate-prerequisites-exist.sh]: Done.

File: api/db-cdr/generate-cdr/validate-prerequisites-exist.sh [hunk 3]
```diff
@@ -8,6 +8,7 @@ export BQ_DATASET=$2        # CDR dataset
...
 export DATA_BROWSER=$4      # data browser flag
```
[the remark this answers is not in the document]
Comment (dolbeew) [reply api/db-cdr/generate-cdr/validate-prerequisites-exist.sh]: Done.

File: api/db-cdr/generate-cdr/validate-prerequisites-exist.sh [hunk 4]
```diff
@@ -8,6 +8,7 @@ export BQ_DATASET=$2        # CDR dataset
...
 
+schema_path=generate-cdr/bq-schemas
 BUCKET="all-of-us-workbench-private-cloudsql"
 TEMP_FILE_DIR="csv"
 CSV_HOME_DIR="cdr_csv_files"
```
[the remark this answers is not in the document]
Comment (dolbeew) [reply api/db-cdr/generate-cdr/validate-prerequisites-exist.sh]: Ah, should've noticed that. Done.

File: api/libproject/devstart.rb [hunk 5]
```diff
@@ -847,6 +847,44 @@ def make_cb_criteria(cmd_name, *args)
...
+  op.add_option(
+    "--bq-dataset [bq-dataset]",
+    ->(opts, v) { opts.bq_dataset = v},
+    "BQ dataset. Required."
+  )
+  op.add_option(
+    "--cdr-version [cdr-version]",
+    ->(opts, v) { opts.cdr_version = v},
+    "CDR version. Required."
+  )
```
[the remark this answers is not in the document]
Comment (dolbeew) [reply api/libproject/devstart.rb]: Done.

File: api/libproject/devstart.rb [hunk 6]
```diff
@@ -847,6 +847,44 @@ def make_cb_criteria(cmd_name, *args)
...
+  op.add_option(
+    "--data-browser [data-browser]",
+    ->(opts, v) { opts.data_browser = v},
+    "Is this run for data browser. Optional - Default is false"
+  )
+  op.add_validator ->(opts) { raise ArgumentError unless opts.bq_project and opts.bq_dataset and opts.cdr_version }
```
[the remark this answers is not in the document]
Comment (dolbeew) [reply api/libproject/devstart.rb]: Done.

File: api/libproject/devstart.rb (hunk 1 above)
Comment (dolbeew) [reply api/libproject/devstart.rb]: Done.

**Review by freemabd — APPROVED** [review]

**Pull request merged**